### PR TITLE
Portfolio Desktop Version

### PR DESCRIPTION
**Pull request by sevinchek**
In this branch I:
- Updated the main, works, about, and contact sections for the desktop version
- Used flexbox and grid for the desktop version
- Updated the README file
- Added Screenshots

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -580,8 +598,374 @@ section > hr {
...
+    margin: 0 0 0 25px;
+    background-color: #3c3a39;
+    border: none;
+    cursor: pointer;
+    display: flex;
+  }
+
+  .desk-img-mail {
+    height: 20px;
+    width: 20px;
```
Comment (DeliceLydia) [inline style.css:657]: - [ ] I would suggest that you use a ( min-width: 1024px) on a desktop and pay attention on how each section behaves

Comment (DeliceLydia) [reply style.css:657]: - [ ] The project section

res

Comment (DeliceLydia) [reply style.css:657]: how the original looks 

orgn

File: style.css [hunk 2]
```diff
@@ -580,8 +598,374 @@ section > hr {
...
+
+  .language,
+  .frameworks,
+  .skills {
+    list-style-type: none;
+    padding: 0;
+    margin-top: -40px;
+  }
+
+  .lang {
```
Comment (DeliceLydia) [inline style.css:928]: - [ ] Skills section

skills

- Original
fx

Comment (DeliceLydia) [reply style.css:928]: - [ ] Flexbox can serve you well here

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @sevinchek, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] Stick to the design as much as you can

_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
Hello @sevinchek,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following point is optional to implement however it's highly recommended!_:blush:

- Kindly include the project starter in the readme file.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Comment by sevinchek** [comment]
> Hello @sevinchek,
> # Congratulations! 🎉
> #### Your project is approved! There is nothing else to say other than... it's time to merge it 💪 💪
> ## Optional Suggestions 💡
> 
> _The following point is optional to implement however it's highly recommended!_😊
> 
>     * Kindly include the project starter in the readme file.
> 
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> Cheers and Happy coding!👏👏👏


Hi @Shaher-11,

Can you give an example of the project starter?

**Comment by Shaher-11** [comment]
HI @sevinchek,
Include a link for the project repo as well as instructions on how to clone the project and get it working locally.

**Comment by sevinchek** [comment]
Ok, thanks @Shaher-11

**Pull request merged**